src/controllers/handlers/error-handler.ts

```diff
@@ -0,0 +1,48 @@
+import { IHttpServerComponent } from '@well-known-components/interfaces'
+import { ErrorResponse, InvalidRequestError, NotFoundError } from '../../types'
+
+function handleError(error: any): { status: number; body: ErrorResponse } {
+  if (error instanceof InvalidRequestError) {
+    return {
+      status: 400,
+      body: {
+        error: 'Bad request',
+        message: error.message
+      }
+    }
+  }
+
+  if (error instanceof NotFoundError) {
+    return {
+      status: 404,
+      body: {
+        error: 'Not Found',
+        message: error.message
+      }
+    }
+  }
+
+  throw error
```

Comment:
perhaps this could return the status 500 json, because I see there is another try catch out of this